File: level1/ex31.js
function solution(phone_number) {
  // 전화번호 길이
  const n = phone_number.length;
  // 앞 (n - 4)자리
  const frontNumber = new Array(n - 4).fill('*').join('');
  // 뒤 4자리
  const backNumber = phone_number.slice(n - 4);
  
  return frontNumber + backNumber;
}

File: level1/ex31.py
def solution(phone_number):
    # 핸드폰 번호 길이
    n = len(phone_number)
    # 앞 (n - 4) 자리 번호
    frontNumber = '*' * (n - 4)
    # 뒤 4 자리 번호
    backNumber = phone_number[-4:]
    
    return frontNumber + backNumber